chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/JLoadTest.java

```diff
@@ -52,12 +59,25 @@ public static Builder builder(Id id, JTestDefinition definition, JLoadProfile lo
         private final JTestDefinition jTestDefinition;
         private final JLoadProfile load;
         private final JTerminationCriteria termination;
-    
+        private List<JLimit> limits;
+
         private Builder(Id id, JTestDefinition jTestDefinition, JLoadProfile load, JTerminationCriteria termination) {
             this.id = id;
             this.jTestDefinition = jTestDefinition;
             this.load = load;
             this.termination = termination;
+            this.limits = new LinkedList<>();
+        }
+
+
+        /**
+         * Set limits for a test.
+         *
+         * @param limits list of {@link JLimit}.
+         */
+        public Builder withLimits(List<JLimit> limits) {
```

Comment:
add please vararg version of this method 
`public Builder withLimits(JLimit limit, JLimit... limits)`